flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

```diff
@@ -567,7 +567,7 @@ public GetClusterStatusResponse getClusterStatus() {
 
 	@Override
 	public int getMaxSlots() {
-		return 0;
+		return -1;
```

Comment:
I suggest to assign this magic number to some `static final` variable with a descriptive name and use it here and in `CliFrontend`, so that the connection is clear.